src/connectors/github_organization.py

```diff
@@ -0,0 +1,251 @@
+                    , value:head_commit::VARIANT head_commit
+                    , value:repository::VARIANT repository
+                    , value:pusher::VARIANT pusher
+                    , value:organization::VARIANT organization
+                    , value:sender::VARIANT sender
+                    , value:action::VARCHAR(256) action
+                    , value:check_run::VARIANT check_run
+                    , value:check_suite::VARIANT check_suite
+                    , value:number::NUMBER(38,0) number
+                    , value:pull_request::VARIANT pull_request
+                    , value:label::VARIANT label
+                    , value:requested_team::VARIANT requested_team
+                    , value:ref_type::VARCHAR(256) ref_type
+                    , value:master_branch::VARCHAR(256) master_branch
+                    , value:description::VARCHAR(256) description
+                    , value:pusher_type::VARCHAR(256) pusher_type
+                    , value:review::VARIANT review
+                    , value:changes::VARIANT changes
+                    , value:comment::VARIANT comment
+                    , value:issue::VARIANT issue
+                    , value:id::NUMBER(38,0) id
+                    , value:sha::VARCHAR(256) sha
+                    , value:name::VARCHAR(256) name
+                    , value:target_url::VARCHAR(8192) target_url
+                    , value:context:: VARCHAR(256) context
+                    , value:state:: VARCHAR(256) state
+                    , value:commit::VARIANT commit
+                    , value:branches:VARIANT branches
+                    , value:created_at::TIMESTAMP_LTZ(9) created_at
+                    , value:updated_at::TIMESTAMP_LTZ(9) updated_at
+                    , value:assignee::VARIANT assignee
+                    , value:release::VARIANT release
+                    , value:membership::VARIANT membership
+                    , value:alert::VARIANT alert
+                    , value:scope::VARCHAR(256) scope
+                    , value:member:VARIANT member
+                    , value:requested_reviewer::VARIANT requested_reviewer
+                    , value:team::VARIANT team
+                    , value:starred_at::TIMESTAMP_LTZ(9) starred_at
+                    , value:pages::VARIANT pages
+                    , value:project_card::VARIANT project_card
+                    , value:build::VARIANT build
+                    , value:deployment_status::VARIANT deployment_status
+                    , value:deployment::VARIANT deployment
+                    , value:forkee::VARIANT forkee
+                    , value:milestone::VARIANT milestone
+                    , value:key::VARIANT key
+                    , value:project_column::VARIANT project_column
+                    , value:status::VARCHAR(256) status
+                    , value:avatar_url::VARCHAR(256) avatar_url
+                FROM data.{base_name}_stream
```

Comment:
Tasks select from streams (or from tables); a pipe should select from a stage and insert into a table.

Reply:
We don't have the stage anymore correct?

Reply:
Ah, we definitely need a stage.

For an example of what the code might look like, check out the aws_vpc_flow_log here: https://github.com/snowflakedb/SnowAlert/pull/260/files#diff-e05b0e58621a6d15bb1884cba631fb54

The basic case of ingesting files from S3 to Snowflake requires a stage, a landing table, and a pipe. The stage is an interface between Snowflake and the external location, and gives Snowflake a file-level view. The pipe operates on files, parses them, and inserts (or copies) the results to the landing table.

The more complicated case is the one for Cloudtrail and Config, where you have the stage, landing table, and pipe, but also a staging table, stream, and ingest task. That's for situations where the file-level view isn't sufficient for ingestion (for example, if one row in the file contains multiple event rows of your logs, e.g. the Records key in a Cloudtrail file). You can't do a flatten inside of a pipe definition, so you have the pipe copy files to the stage, and then do the flattening inside of a task operating on a stream tied to the pipe, and the task copies from the staging table to the landing table.

Does that make sense? I'm happy to jump on a quick call with you to go over the code in person if that would be productive, or if the model I'm describing isn't clear.